ui/components/AutoComplete.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/prop-types */
+import * as React from "react";
+import Downshift from "downshift";
+
+
+export const AutoComplete = ({items}) => {

```

Comment:
You need to take another prop here, `handleSelectionChange`. This can be passed to the `onChange` function in DownShift component.